tests/tests_lvm_pool_shared.yml

```
@@ -0,0 +1,115 @@
+---
+- name: Test LVM shared pools
+  hosts: all
+  become: true
+  vars:
+    storage_safe_mode: false
+    storage_use_partitions: true
+    mount_location1: '/opt/test1'
+    volume1_size: '4g'
+
+  tasks:
+    - name: Create cluster
+      ansible.builtin.include_role:
+        name: linux-system-roles.ha_cluster
```

```suggestion
        name: fedora.linux_system_roles.ha_cluster
```
For any system role other than the storage role, use `fedora.linux_system_roles.$rolename`.  https://linux-system-roles.github.io/documentation/role-requirements.html
For the storage role, use `linux-system-roles.storage` - see `tests/roles/linux-system-roles.storage/` and the symlinks there.

---

It's a little more complicated to use the ha_cluster role for testing - see https://github.com/linux-system-roles/mssql/blob/main/tests/tests_configure_ha_cluster.yml

https://github.com/linux-system-roles/mssql/blob/main/tests/tests_configure_ha_cluster.yml#L165C30-L165C30
```yaml
        - name: Set up test environment for the ha_cluster role
          include_role:
            name: fedora.linux_system_roles.ha_cluster
            tasks_from: test_setup.yml
```

and then cleanup too https://github.com/linux-system-roles/mssql/blob/main/tests/tests_configure_ha_cluster.yml#L981